packages/eslint-config-eslint/formatting.js

```diff
@@ -1,123 +1,135 @@
 "use strict";
 
+const rules = {
```

Comment:
What's the purpose of this change?

Reply:
It's so that we can use named imports in ESM files:

```ts
import { rules } from "eslint-config-eslint/formatting";

export default [...eslintConfigESLint, rules] satisfies Linter.Config[];
```

Reply:
In an ESM file we will be able to do:

```ts
import { rules } from "eslint-config-eslint/formatting";

export default [...eslintConfigESLint, rules] satisfies Linter.Config[];
```

Without the provided changes, this would fail.

Reply:
is it supported and documented somewhere? As far as I can remember, cjs should be treated as a default export.

Reply:
it's very helpful, thanks for the explanation!

I agreed with @mdjermanovic said in https://github.com/eslint/eslint/issues/19110#issuecomment-2460723401.
it's explicitly stated the default export usage, and it's the officially recommended; we do not guarantee that non-recommended usage will work.